docs/api/README.md

```diff
@@ -80,3 +82,53 @@ LTS (Focal Fossa), then we would look for the artifact at:
 |name|string|`"rl_drop_count_map"`|The name of the map where metrics are stored|
 |key|number|0|The index in the map specified by `name` where metrics are stored|
 |aggregator|string|scalar|The type of metrics aggregation to use for the configured metric sampling interval. Supported values are `"scalar"`, `"max-rate"`, and `"avg"`.|
+
+
+
+
+# Add API 
+You can Refer to above documentation.
```

Comment:
Does this mean the JSON is the same as for the Update API?  If so, say so.

Reply:
yes

Reply:
```suggestion
The JSON is the same as for the Update API.  Refer to above documentation.
```